CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+Thank you for considering giving feedback or contributing to this project.
+
+There are two ways you can contribute to this repository:
+
+1. **Creating or commenting on issues**: If you spot a mistake or notice inconsistencies, please feel free to create a new issue [here](https://github.com/scunning1975/mixtape/issues). You are also welcome to read through the issues submitted by others and to contribute to the discussion.
+1. **Proposing changes**: If you are familiar with how GitHub works (or are willing to try), you can submit a pull request with any additions or modifications to the files in this repository.
+
+For more details on contributing via GitHub, read through the sections below.
+
+## Contributing through GitHub
+
+To open issues or pull requests, you will need to create an account on <https://github.com/>. For pull request, you will also need to have [set up Git](https://happygitwithr.com/install-git.html) on your local machine.
+
+### Creating or commenting on issues
+
+Issues are places to track ideas, enhancements, tasks, and to report errors or bugs. To create a new issue:
+
+1. Go to [the issues tab](https://github.com/scunning1975/mixtape/issues).
+1. Search the already posted issues to see if the feedback you are about to give is already posted.
+1. If your feedback has not already been brought up, please press the green *New Issue* button to post a new issue.
```

Comment:
I think it should be clearer that folks should create an issue if one doesn't exist before creating a PR